lib/active_admin.rb

```diff
@@ -35,6 +35,7 @@ module ActiveAdmin
   autoload :Event,                    'active_admin/event'
   autoload :FormBuilder,              'active_admin/form_builder'
   autoload :Inputs,                   'active_admin/inputs'
+  autoload :Localizers,               'active_admin/localizers'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.